### PR TITLE
Update prepare-mattermost-database.rst

### PR DESCRIPTION
**Pull request by rohan-kapse**
Added the missing command to change owner to the user.

### CODE REVIEW

File: source/install/prepare-mattermost-database.rst [hunk 1]
```diff
@@ -54,8 +54,16 @@ To set up a PostgreSQL database for use by the Mattermost server:
       :class: mm-code-block 
 
         postgres=# GRANT ALL PRIVILEGES ON DATABASE mattermost to mmuser;
+                  
 
-  b. Grant access to objects contained in the specified schema by running: 
+  b. Grant the user to change the owner of a database to a user mmuser by running:
```
Comment (cwarnermm) [inline source/install/prepare-mattermost-database.rst:59]:
```suggestion
  b. Grant the user to change the owner of a database to a user ``mmuser`` by running:
```

File: source/install/prepare-mattermost-database.rst [hunk 2]
```diff
@@ -206,4 +214,4 @@ To set up a PostgreSQL database for use by the Mattermost server:
 
 The PostgreSQL interactive terminal starts. To exit the PostgreSQL interactive terminal, type ``\q`` and press :kbd:`Enter` on Windows or Linux, or :kbd:`↵` on Mac.
 
-When the PostgreSQL database is installed, and the initial setup complete, you can install the Mattermost server.
\ No newline at end of file
+When the PostgreSQL database is installed, and the initial setup complete, you can install the Mattermost server.
```
Comment (cwarnermm) [inline source/install/prepare-mattermost-database.rst:217]:
```suggestion
When the PostgreSQL database is installed, and the initial setup is complete, you can install the Mattermost server.
```

**Review by cwarnermm — APPROVED** [review]
Thank you, @rohan-kapse!

**Comment by rohan-kapse** [comment]
happy to make a small contribution @cwarnermm : ) !!!

**Comment by cwarnermm** [comment]
@rohan-kapse Small, perhaps, but definitely mighty!!

**Pull request merged**